dali/operators/reader/video_reader_op.cc

```diff
@@ -26,63 +26,83 @@ namespace dali {
 DALI_REGISTER_OPERATOR(VideoReader, VideoReader, GPU);
 
 DALI_SCHEMA(VideoReader)
-  .DocStr(R"code(
-Load and decode H264 video codec with FFmpeg and NVDECODE, NVIDIA GPU's hardware-accelerated video decoding.
-The video codecs can be contained in most of container file formats. FFmpeg is used to parse video containers.
-Returns a batch of sequences of `sequence_length` frames of shape [N, F, H, W, C] (N being the batch size and F the
-number of frames). Supports only constant frame rate videos.)code")
+  .DocStr(R"code(Loads and decodes the H264 video codec with FFmpeg and NVDECODE, which is
+the hardware-accelerated video decoding feature in the NVIDIA(R) graphical process unit (GPU).
```

Comment:
```suggestion
  .DocStr(R"code(Loads and decodes video files using FFmpeg and NVDECODE, which is
the hardware-accelerated video decoding feature in the NVIDIA(R) graphics process unit (GPU).
```
!!!!!!!!

Reply:
although I don't think we even need to spell out what a GPU. It sounds to start like a sales pitch.

Reply:
Done